core/commands/add.go

```diff
@@ -37,6 +37,7 @@ const (
 	progressOptionName    = "progress"
 	trickleOptionName     = "trickle"
 	wrapOptionName        = "wrap-with-directory"
+	wrapPathName          = "name"
```

Comment:
So, as discussed in the issue, this is not exactly related to the `-w` (wrap) flag, is a more generic issue of assigning names to a unnamed file content (such as the one received from `stdin`). Let's change the name of this option to reflect that, e.g., `wrapPathName` -> `pathName`.